### ISSUE TITLE
CuckooError: Unable to import package "modules.packages.None", does not exist.

### ISSUE THREAD

**Issue by zhaoyaochen** [cuckoosandbox/cuckoo#1895]
@@[sample.zip](https://github.com/cuckoosandbox/cuckoo/files/1389671/sample.zip)
i want to analysis SWF file, but caught a exception:
2017-10-16 23:17:20,554 [cuckoo.core.resultserver] DEBUG: LogHandler for live analysis.log initialized.
2017-10-16 23:17:21,104 [cuckoo.core.guest] WARNING: cuckoo11: analysis caught an exception
**_Traceback (most recent call last):
  File "C:/tmpngphkx/analyzer.py", line 794, in <module>
    success = analyzer.run()
  File "C:/tmpngphkx/analyzer.py", line 579, in run
    "not exist.".format(package_name))
CuckooError: Unable to import package "modules.packages.None", does not exist._**

2017-10-16 23:17:21,172 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Sniffer

**Comment by jbremer** [comment]
Thanks! Going to take a look soon.

**Comment by n00b101** [comment]
I have the same issue.... can anyone sort this out?

2017-10-31 04:58:57,892 [cuckoo.core.guest] WARNING: cuckoo1: analysis caught an exception
Traceback (most recent call last):
  File "C:/tmpigxne5/analyzer.py", line 794, in <module>
    success = analyzer.run()
  File "C:/tmpigxne5/analyzer.py", line 579, in run
    "not exist.".format(package_name))
CuckooError: Unable to import package "modules.packages.None", does not exist.

2017-10-31 04:58:57,992 [cuckoo.core.plugins] ERROR: Unable to stop auxiliary module: Sniffer
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/core/plugins.py", line 162, in stop
    module.stop()
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/auxiliary/sniffer.py", line 154, in stop
    (out, err, faq("permission-denied-for-tcpdump"))
CuckooOperationalError: Error running tcpdump to sniff the network traffic during the analysis; stdout = '' and stderr = 'tcpdump: vboxnet0: That device is not up\n'. Did you enable the extra capabilities to allow r

**Comment by zhaoyaochen** [comment]
@n00b101 you could run this command: VBoxManage hostonlyif ipconfig vboxnet0 --ip 192.168.56.1
and assure you vboxnet0 is up.
https://cuckoo.sh/docs/installation/host/requirements.html#installing-tcpdump this is tcpdump's solution

**Comment by n00b101** [comment]
still got the same error, my vbox is configured as hostonly.

**Comment by zhaoyaochen** [comment]
Traceback (most recent call last):
File "C:/tmpigxne5/analyzer.py", line 794, in 
success = analyzer.run()
File "C:/tmpigxne5/analyzer.py", line 579, in run
"not exist.".format(package_name))
CuckooError: Unable to import package "modules.packages.None", does not exist.

don't care this error

**Comment by doomedraven** [comment]
try
```
groupadd pcap
usermod -a -G pcap cuckoo
chgrp pcap /usr/sbin/tcpdump
setcap cap_net_raw,cap_net_admin=eip /usr/sbin/tcpdump
```

**Comment by RealLinkers** [comment]
I am getting the same error currently only with .zip, .7z, rar files.

**Comment by infinitesecure** [comment]
Same problem here.: Getting this error on .zip, .7z, .rar files.

**Comment by Wulluby** [comment]
Same on .rar files
69f67c28b26bab5eb3dfb5421f7ef691
d04fc87968b9085596d87ee6f1df8667 

And on .zip 
323c398a2c08819ce3ad8bbe19c42da4 

I think this might be connected to something referenced in the Analysis Packages section of the docs, "specify the name of the file contained in the archive to execute. If none is specified, Cuckoo will try to execute sample.exe."

I was getting similar to the following when submitting these,
Traceback (most recent call last):
File "C:/tmpigxne5/analyzer.py", line 794, in
success = analyzer.run()
File "C:/tmpigxne5/analyzer.py", line 579, in run
"not exist.".format(package_name))
CuckooError: Unable to import package "modules.packages.None", does not exist.

I noticed when submitting the zip through the web interface though that there were 2 possibilities for submission, when selecting the exe inside the zip, the exe package was autoselected and analysis was a success. As the contents of the rar are not visible I used unrar l <filename> to get the filenames from inside from where I found <filename.exe>, then when submitting I added into extra option name=file & value=<filename.exe> but this errored too.

Late here now, will try again tomorrow for the rar files.
